task/03-date-tasks.js

```diff
@@ -76,7 +80,12 @@ function isLeapYear(date) {
  *    Date(2000,1,1,10,0,0),  Date(2000,1,1,15,20,10,453)   => "05:20:10.453"
  */
 function timeSpanToString(startDate, endDate) {
-   throw new Error('Not implemented');
+
+   const differenceDate = new Date(endDate - startDate).toISOString();
+   const days = differenceDate.slice(8,10) - 1;
+   const hours = differenceDate.slice(11,13);
+   
+   return  (days > 0) ? ( (days * 24 + +hours) + differenceDate.slice(13,23) ) : differenceDate.slice(11,23);
 }
 
```

Comment:
seems it works but could you use get(Hours/Minutes/Seconds/Milliseconds) methods